src/Libraries/CoreNodeModels/HigherOrder/MapCombineLacing.cs

```diff
@@ -141,6 +160,11 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
     [AlsoKnownAs("DSCore.LaceShortest", "DSCoreNodesUI.HigherOrder.LaceShortest")]
     public class LaceShortest : CombinatorNode
     {
+        [JsonConstructor]
+        private LaceShortest(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) :base(inPorts, outPorts) { }
+
+        public LaceShortest() : base() { }
```

Comment:
API Change : Public default constructors have been added for `CombinatorNode` types to facilitate serialization.